DataStructures/Lists/SinglyLinkedList.java

```diff
@@ -61,7 +61,9 @@ public void deleteHead() {
             throw new RuntimeException("The list is empty!");
         }
 
+        Node destory = head;
```

Comment:
Typo on destory, I believe you're aiming for destroy

Reply:
Thanks!